### PR TITLE
Helps 4369 add browser aptitude to vs code npx

### PR DESCRIPTION
**Pull request by oelumezeolive**
###

- Browser Aptitude functionality.

[HELPS-4369]: https://crosschx.atlassian.net/browse/HELPS-4369?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by jonathan-olive** [comment]
You're going to want to add this aptitude to the npx tool in the`create-loop` folder
i.e. 
```
    choices: [
     { title: 'Browser', value: 'browser'},
      { title: 'Clipboard', value: 'clipboard' },
      { title: 'Document', value: 'document' },
      { title: 'Filesystem', value: 'filesystem' },
      { title: 'Keyboard', value: 'keyboard' },
      { title: 'Network', value: 'network' },
      { title: 'Process', value: 'process' },
      { title: 'UI', value: 'ui' },
      { title: 'User', value: 'user' },
      { title: 'Window', value: 'window' },
    ],
```

**Review by jonathan-olive — COMMENTED** [review]
Missing a few files:
in `loop-templates`
`index.test.ts.squirrelly` in config aptitude
`index.ts.squirrelly` in config aptitude

`index.ts.squirrelly` in config whisper

in `Vscode-extension`
`Changelog.md`
in `package.json` make sure that "@oliveai/loop-templates": "../loop-templates",

Not sure if it's me, but for some reason not able to get anything to show when creating a loop. You might want to `git stash` and create a clean branch from features to avoid some of Ryan's commits. Or it could be caused by some of the missing files above. 
@aleciberg-olive Can you help verify this?

File: vscode-extension/src/createLoopForm.html [hunk 1]
```diff
@@ -194,6 +194,17 @@ <h2>Create Loop</h2>
...
+            <td>
+              <input
+                type="checkbox"
+                id="aptitudesBrowser"
+                name="aptitudes"
+                value="browser"
+              />
+              <label for="aptitudesBrowser">Browser</label>
+            </td>
+          </tr>
```
[the remark this answers is not in the document]
Comment (aleciberg-olive) [reply vscode-extension/src/createLoopForm.html:207]: Yeah I think you're correct Jonny, might need to stash and roll back

**Comment by aleciberg-olive** [comment]
The empty files in `create-loop/src/aptitudes/browser/ `can be removed, all of those should live in the loop-templates.

File: loop-templates/templates/src/aptitudes/browser/browserExample.ts.squirrelly [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    */
+    
+    // navDetails.url === URL_TO_OPEN: true
+    // navDetails.tabId === newTabId: true
+
+    // Cancel the listener when a specific URL is opened by the user
+    if (navDetails.url === URL_TO_OPEN) {
+        browserListener.cancel();
+    }
+})
```
Comment (jonathan-olive) [inline loop-templates/templates/src/aptitudes/browser/browserExample.ts.squirrelly:26]: const run = async () => {
  const navDetails = await config.browserListener();

  await BrowserWhisper.show(navDetails);
};

export default { run };

Comment (jonathan-olive) [reply loop-templates/templates/src/aptitudes/browser/browserExample.ts.squirrelly:26]: Change to browserListener
Add tests

**Pull request closed without merging**